spec/helpers/application_helper_spec.rb

```diff
@@ -25,4 +25,27 @@
       end
     end
   end
+
+  describe "title" do
+
+    context "nil input" do
+      it 'returns nil' do
+        expect(title).to be nil
+      end
+    end
+
+    context "non-nil input" do
+      it 'returns nil for an empty string' do
+        expect(title('')).to be nil
+      end
+
+      it 'returns the title for a non-empty string' do
+        helper.title('Courses')
+        expect(helper.instance_variable_get(:@view_flow).
```

Comment:
This will leave this test quite brittle, if the name of this variable is ever changed this test will break.

I did a bit of research on testing content_for just now. I don't really like using content for in the tests, but this is probably the cleanest way to test it without stubbing out content_for.

```
it 'returns correctly formatted title' do
      helper.title('hello')
      expect(helper.content_for(:title)).to eql('hello | The Odin Project')
    end
```